cmd/scollector/collectors/snmp_ifaces.go

```diff
@@ -64,11 +77,23 @@ func c_snmp_ifaces(community, host string) (opentsdb.MultiDataPoint, error) {
 	if err != nil {
 		return nil, err
 	}
+	t, err := snmp_subtree(host, community, ifType)
```

Comment:
can we fix up some of the one letter variable names in here? k and v I don't mind but a,n,t should all be more descriptive. `snmpNames`, `snmpTypes` perhaps? `ifTypes`?